weblate/templates/format-translation.html

```diff
@@ -1,11 +1,23 @@
+{% load i18n %}
 {% if simple %}
 <span lang="{{ language.code }}" dir="{{ language.direction }}">{{ items.0.content }}</span>
 {% else %}
 <div class="list-group">
   {% for item in items %}
   <div class="list-group-item">
     {% if item.title %}
-    <h5 class="list-group-item-heading"><strong>{{ item.title }}</strong></h5>
+    <h5 class="list-group-item-heading">
+      <strong>{{ item.title }}</strong>
+      {% if copy_to %}
+        <button class="btn btn-xs btn-default copy-text"
```

Comment:
To be consistent, this should have also `pull-right flip` classes. Similarly for the other button code, maybe they could use shared template?